### DISCUSSION TITLE
What is the difference between Eclipse Theia and Coder Code Server?

### DISCUSSION THREAD

**Discussion by viveksh1** [coder/code-server#3285] [Q&A] [answered]
What is the difference between Eclipse Theia and Coder Code Server?

**Comment by jsjoeio** [comment, marked as answer]
I can explain it but @bpmct may have already in the past. Let me check with him first.

**Reply by ghost** [reply]
https://github.com/cdr/code-server/blob/main/docs/FAQ.md#differences-compared-to-theia

Related
- Issue #1756
- Pull Request #1767

**Reply by jsjoeio** [reply]
^Even better - thanks @AurangAsif.

And for the explanation between Coder and code-server, see @bpmct's comment here: https://github.com/cdr/code-server/issues/3094#issuecomment-817389874
